lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -38,6 +39,18 @@ fi
 echo_summary "Performing sanity checks..."
 
 
+### Check if repositories are prepared for the upgrade
+# We need to ensure that Cloud5 main repository and update repos are in place
+
+check_media_content \
+    Cloud \
+    /srv/tftpboot/suse-11.3/repos/Cloud \
+    1558be86e7354d31e71e7c8c2574031a
+
+check_repo_tag summary 11.3 SUSE-Cloud-5-Pool      'SUSE Cloud 5'
+check_repo_tag repo    11.3 SUSE-Cloud-5-Updates   'updates://zypp-patches.suse.de/autobuild/SUSE_CLOUD/5/update/x86_64'
```

Comment:
Pretty sure we need to use REPOS_SKIP_CHECKS for now by default, as this likely doesn't work at the moment :/